test/inplace/inplace.mk

```diff
@@ -0,0 +1,7 @@
+
+# Installs Istio 1.2 using default install yaml, install canary, verify canary starts.
+test-inplace:
+	kubectl apply -k kustomize/istio-1.2/default --prune -l release=istio
+	kubectl apply -k kustomize/istio-canary --prune -l release=istio-canary
```

Comment:
won't the prune here delete the existing install?

Reply:
No - prune is looking for any resource with 'release=istio-canary' ( and there will be none in this case ) and if it doesn't exist in the new template it'll remove it.

If later we deploy again istio-canary, with an option that removes one of the deployments - when prune is called it will delete the missing deployment.